project-base/templates/Front/Content/Category/panel.html.twig

```diff
@@ -1,12 +1,15 @@
 {% if categoriesWithLazyLoadedVisibleChildren|length > 0 %}
-    <ul class="js-category-list list-menu dont-print {% if isFirstLevel %}list-menu--root{% endif %}" {% if isFirstLevel %}id="js-categories"{% endif %}>
+
+    <ul class="js-category-list list-menu dont-print {% if isFirstLevel %}list-menu--root{% endif %}
+    {% if categoriesWithLazyLoadedVisibleChildren[0].category.level > 2 %}list-menu--dropdown{% endif %}
+    " {% if isFirstLevel %}id="js-categories"{% endif %}>
```

Comment:
Please format this in better way, this is hard to read. Refactor classes to variables etc.

Reply:
I tried to format i better way. But actually dont know how use variables - I dont think it will be better. Can you draw your idea how to do it?